Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -298,3 +291,21 @@ extension Module  {
         return buildSettings
     }
 }
+
+extension Plist {
+    // FIXME: This is only internal for unit testing.
+    /// Quotes the strings in the Plist.
+    func quoteStrings() -> Plist {
+        switch self {
+        case .string(let str):
+            return .string("\"" + str + "\"")
```

Comment:
When I say "still needs to be escaped", I'm not talking about whether or not Xcode expects a certain level of quoting.  I mean that in a very local sense, if `str` is equal to `"` here, then you will end up with an output (of just this function) that is `"""`.  That's broken.